tetrateci/fetch_all_tags.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+# we first fetch all the tags from origin and then from upstream
+# the tag should be in form of number.number.number
+git remote add upstream https://github.com/istio/istio
+git fetch --tags origin
+git tag -l | grep -E "^[0-9]+.[0-9]+.[0-9]+$" > /tmp/oldtags
+git fetch --tags upstream
+git tag -l | grep -E "^[0-9]+.[0-9]+.[0-9]+$" > /tmp/newtags
+
+echo "print oldtags"
+cat /tmp/oldtags
+echo "print newtags"
+cat /tmp/newtags
+
+# then compare the list and pick the ones unique to the second list
+tags=$(comm -13 /tmp/oldtags /tmp/newtags)
+
+git config user.name github-actions
+git config user.email github-actions@github.com
+
+for tag in $tags; do
+    # the branch names are suffixed with the first 2 numbers in the version
+    branch=$( echo $tag | cut -d. -f1,2 )
+    if [[ ! $(git rev-parse --verify --quiet origin/tetrate-release-$branch) ]]; then
+        # create the tetrate release branch if it doesn't exist with the workflows
+        git checkout tetrate-workflow
```

Comment:
git checkout -b tetrate-release-$branch  tetrate-workflow 

instead of the two commands

Reply:
Tried that first, didn't work so switched to this

Reply:
shell into the runner and check it out.

Reply:
https://github.com/tetratelabs/getistio/issues/78